dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
+ * (c) 2015 Vokal LLC
+ * https://github.com/vokal/angular-svg-radial
+ */
+angular.module("svgRadial", []).directive("radialProgress", function() {
+    "use strict";
+    return {
+        restrict: "E",
+        transclude: true,
+        replace: true,
+        template: '<div class="radial-progress">' + '<div class="base"></div>' + '<div class="radial">' + '<div class="piece"></div>' + '<div class="piece"></div>' + "</div>" + '<div class="transclude" ng-transclude></div>' + "</div>",
```

Comment:
@Tathanen This is uglify's output when you set `beautify: true`. It allows us to tack on the license header and keep the versioning updated but still output a file that is readable should someone want to swap it out for the `.min` file.

Eventually the template will be in a separate file and included via ngtemplates.